src/Client/ClientBase.cpp

```diff
@@ -2190,6 +2190,8 @@ bool ClientBase::executeMultiQuery(const String & all_queries_text)
     const char * this_query_begin = all_queries_text.data() + test_tags_length;
     const char * this_query_end;
     const char * all_queries_end = all_queries_text.data() + all_queries_text.size();
+    const char * last_query_begin = this_query_begin;
+    UInt64 current_line = std::count(all_queries_text.c_str(), this_query_begin, '\n') + 1;
```

Comment:
Quadratic complexity.

Reply:
I dont get this comment, std::count() has complexity of O(n), this + std::count() in loop, wouldn't change time complexity of this loop.